src/components/Disputes/DisputeCard.js

```diff
@@ -42,38 +35,23 @@ function DisputeCard({ dispute, selectDispute }) {
             ${textStyle('body1')}
           `}
         >
-          Dispute
-          <strong> #{id}</strong>
+          <strong> Dispute #{id}</strong>
         </h3>
         <DisputeText
-          text={description}
+          text={metadata}
           css={`
             overflow: hidden;
             ${textStyle('body2')};
-            color: ${theme.contentSecondary};
+            color: ${theme.content};
             line-height: ${27}px; // 27px = line-height of textstyle('body1')
-            height: ${27 * 2}px; // 27px * 2 = line-height * 2 lines
+            height: ${27 * 3}px; // 27px * 3 = line-height * 3 lines
             display: -webkit-box;
             -webkit-box-orient: vertical;
-            -webkit-line-clamp: 2;
+            -webkit-line-clamp: 3;
           `}
         />
-        <IdentityBadge entity={creator} badgeOnly />
       </div>
-      <DisputeDetails labelColor={theme.contentSecondary}>
-        <div>
-          <span>Reward</span>
-          <span>{rewardAmount} DAI</span>
-        </div>
-        <div>
-          <span>Collateral staked</span>
-          <span>{stakedAmount} ANJ</span>
-        </div>
-        <div>
-          <span>Term Date</span>
-          <span>{termDate}</span>
-        </div>
-      </DisputeDetails>
+      <DisputePhase phase={currentPhase} />
```

Comment:
@rperez89 With the default dispute, I get an empty `currentPhase` (`undefined`), do you know why this is the case? Even if it shouldn’t happen, maybe the UI could adapt to it by not rendering the component?

Reply:
Hey! remember that many things regarding to the phase and transition were fixed [here](https://github.com/aragon/court-dashboard/blob/1e42271a9fa9dd08b3bfce9e36a6d473b283dfeb/src/hooks/useDisputes.js#L14). Now we shouldn't have an undefined phase. This PR is one of the first so many things were fixed in anothers PR's . But taking a note here to check in the most recent PR if that is heppening

Reply:
Yes you’re right, I don’t have the issue anymore in https://github.com/aragon/court-dashboard/pull/39